INSTALL.md

```diff
@@ -36,6 +36,22 @@ make bash
 
 All the source and dependencies for this project will be in place, making it possible to run porta and the tests from inside the container. See [Run Porta](#run-porta)
 
+#### Running the application
+
+If you want to run the application using docker, you can use the docker compose file to start all dependencies.
+
+In the first time, you need to setup the database. To configure it, run: (the Master and User password will be `p`)
+
+```
+MASTER_PASSWORD=p USER_PASSWORD=p make dev-setup
```

Comment:
Do we need to pass `MASTER_PASSWORD` and `USER_PASSWORD` again if the `run` target will set them to these same default values?

Reply:
it is not needed. Just added here in the docs as an example that it can be overwritten.

Reply:
About the name of the task, `dev-setup` implies it is the setup of a development env. But the point of our Dockerfile is to have a production app running, no?

Reply:
@josemigallas the Dockerfile itself yes. But this commit also includes the `docker-compose` file that is used for development. So the `dev-*` tasks are the one used for running locally